packages/service/src/service.ts

```diff
@@ -26,8 +31,13 @@ const auth = (req, resp, next) => {
 };
 
 passport.use(
-  new LocalStrategy(async function (username, _password, done) {
-    const user = await createUser({ name: username });
+  new LocalStrategy(async function (username, password, id, done) {
+    const user = (
+      await validateUser({ name: username, password: password, id: id })
```

Comment:
I don't know where this `id` field is coming from. Validation should only need a username and password